src/main.rs

```diff
@@ -1,24 +1,27 @@
 /// Executable for proton_cli
-
-extern crate proton_cli;
-extern crate git2;
 extern crate rustc_serialize;
+extern crate proton_cli;
 extern crate docopt;
 
 use std::env;
 use std::path::Path;
 use docopt::Docopt;
-use proton_cli::Error;
+
+use proton_cli::error::Error;
+use proton_cli::PermissionEnum;
+use proton_cli::utils;
 
 
 const USAGE: &'static str = "
 Command-line interface for Proton
 
 Usage:
-  ./proton init <folder>
-  ./proton new-user <name> <public-key>
+  ./proton init <folder> <public-key>
+  ./proton new-user <admin-key> <name> <public-key>
+  ./proton new-sequence <admin-key> <name> <music-file>
   ./proton id-user <private-key>
-  ./proton new-sequence <name> <music-file>
+  ./proton list-permissions
+  ./proton set-permission <admin-key> (add | remove) <name> <permission> [<target>]
```

Comment:
I like the idea of combining the permissions for now. If we feel they belong apart in the future, we can separate them then. Not sure about the name though; Manage sounds kinda vague. I wouldn't mind just calling it Admin, since that's what it basically is. Still would have the confusion though. What if the project admin's key was instead called project-key?